src/components/gds-media-card/gds-media-card.tsx

```diff
@@ -99,56 +135,16 @@ export class GdsMediaCard {
       </gds-card>
     )
 
-    const isNumeric = x => !isNaN(x) && isFinite(x)
-
-    const objectFitX =
-      isNumeric(this.superimposedLeft) && isNumeric(this.superimposedRight)
-        ? 'center'
-        : isNumeric(this.superimposedLeft)
-        ? 'left'
-        : isNumeric(this.superimposedRight)
-        ? 'right'
-        : 'center'
-
-    const objectFitY =
-      isNumeric(this.superimposedTop) && isNumeric(this.superimposedBottom)
-        ? 'center'
-        : isNumeric(this.superimposedTop)
-        ? 'top'
-        : isNumeric(this.superimposedBottom)
-        ? 'bottom'
-        : 'center'
-
-    // Optional superimposed image.
-    // Note: Superimpose needs to be outside of card because it has overflow hidden.
-    const superimposed = this.superimposedUrl && (
-      <div class="superimposed">
-        <div
-          class="superimposed-image"
-          style={{
-            top: this.superimposedTop && `${this.superimposedTop * -1}px`,
-            bottom: this.superimposedBottom && `${this.superimposedBottom * -1}px`,
-            left: this.superimposedLeft && `${this.superimposedLeft * -1}px`,
-            right: this.superimposedRight && `${this.superimposedRight * -1}px`,
-          }}>
-          <img
-            src={this.superimposedUrl}
-            style={{
-              'object-position': `${objectFitY} ${objectFitX}`,
-            }}
-          />
-        </div>
-      </div>
-    )
-
-    // Container for card and superimpose image
+    // Container for card
     const mediaCard = (
       <div
         class="media-card"
         style={{
-          paddingTop: this.superimposedTop && `${this.superimposedTop}px`,
+          '--superimposed-top': this.superimposedTop ? `${this.superimposedTop}px` : '0px',
+          '--superimposed-bottom': this.superimposedBottom ? `${this.superimposedBottom}px` : '0px',
+          '--superimposed-left': this.superimposedLeft ? `${this.superimposedLeft}px` : '0px',
+          '--superimposed-right': this.superimposedRight ? `${this.superimposedRight}px` : '0px',
         }}>
-        {superimposed}
```

Comment:
Actually this could be moved back, but it doesnt matter for now, neither would support % margins.